docs/application/dotnet/guides/opentk/overview.md

```diff
@@ -26,43 +20,32 @@ The OpenTK app offers the following features:
 
 ## Key Concepts
 
-To use OpenTK in your application, you must familiarize yourself with the following OpenTK key concepts:
+To use OpenTK in your application, it helps you understand the following OpenTK key concepts:
+
+-   `TizenGameApplication`: The application lifecycle manager and base class of OpenTK app for Tizen.
+
+    `TizenGameApplication` is integrated with the Tizen Application Framework. Therefore, you can get the events from the Tizen Application Framework. The OpenTK app implements virtual lifecycle functions to customize their own app flow.  For example, `OnCreate`, `OnPause`, `OnResume`, `OnTerminate`, `OnAppControlReceived`, `OnDeviceOrientationChanged`, and so on.
 
--   `TizenGameApplication`: It is the application lifecycle manager and base class of OpenTK app for Tizen. `TizenGameApplication` is integrated with the Tizen Application Framework. Therefore, you can get the events from the Tizen Application Framework.
-    The OpenTK app implements virtual lifecycle functions to customize their own app flow.
-    For example, `OnCreate`, `OnPause`, `OnResume`, `OnTerminate`, `OnAppControlReceived`, `OnDeviceOrientationChanged`, and so on. 
-    
     For more information, see [Applications](../applications/ui-app.md).
-    
--   `Window`: It is an attribute of TizenGameApplication, and provides window related attributes, which include `X`, `Y`, `Width`, `Height`, `Title`, `WindowInfo`, and so on. 
 
-    It also provides **Events** and **OpenGL** context related functions, which includes the following:
-    
+-   `Window`: An attribute of TizenGameApplication.  It provides window related attributes, which include `X`, `Y`, `Width`, `Height`, `Title`, `WindowInfo`, and so on.
```

Comment:
Please add code tag for TizenGameApplication, it is my mistake.
TizenGameApplication -> `TizenGameApplication`

Reply:
Got it.